### PR TITLE
Add command output configuration

### PR DESCRIPTION
**Pull request by aledeg**
Before, the StreamOutput object was instanciated with default options.
That will cause some problems when testing the output on terminal
supporting decorated outputs.
Now, it is possible to configure the output to remove the decoration and
choosing the verbosity level.

See #180

### CODE REVIEW

File: Test/WebTestCase.php [hunk 1]
```diff
@@ -119,7 +119,19 @@ protected function runCommand($name, array $params = array(), $reuseKernel = fal
         $input->setInteractive(false);
 
         $fp = fopen('php://temp/maxmemory:'.$this->maxMemory, 'r+');
-        $output = new StreamOutput($fp);
+
+        $commandVerbosity = StreamOutput::VERBOSITY_NORMAL;
+        if ($this->getContainer()->hasParameter('liip_functional_test.command_verbosity')) {
```
Comment (lsmith77) [inline Test/WebTestCase.php]: should we not make these properties instead which are set with the configured defaults? this way we can override them on a per test basis

Comment (aledeg) [inline Test/WebTestCase.php]: That's a good idea. But we could even have both.
Could you guide me on how to set new properties in that case? I am not sure were to inject those properties.
Thanks

Comment (aledeg) [inline Test/WebTestCase.php]: I figured how to do that :)

**Comment by lsmith77** [comment]
thx

**Comment by aledeg** [comment]
My pleasure

**Pull request merged**